aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/featuretest/generators/SerializationTools.kt

```diff
@@ -48,6 +48,16 @@ fun writeFile(json: String, dirName: String, fileName: String) {
     println("File written in ${directory.absolutePath}")
 }
 
+fun JsonGenerator.cleanDirectory() {
+    val directory = File("$basePath")
+    if (directory.exists()) {
+        val children = directory.list()
+        for (i in children.indices) {
+            File(directory, children[i]).delete()
+        }
+    }
```

Comment:
does this delete files in subdirs also?
how about just using `directory.deleteRecursively()`?

Reply:
Agreed. Changed.